modules/jupyter-widget/src/utils.js

```diff
@@ -60,3 +61,54 @@ export function initDeck({mapboxApiKey, container, jsonInput}, onComplete, handl
     });
   });
 }
+
+function getTooltip(pickedInfo) {
+  if (!pickedInfo.picked) {
+    return null;
+  }
+  return {
```

Comment:
Cache the last result so we don't repeat the calculation when cursor moves over the same object.